packages/eas-cli/src/commands/branch/view.ts

```diff
@@ -122,8 +126,7 @@ export default class BranchView extends EasCommand {
     );
 
     if (jsonFlag) {
-      Log.log(JSON.stringify({ ...UpdateBranch, updates }));
-      return;
+      printJsonOnlyOutput({ ...UpdateBranch, updates });
```

Comment:
Isn't the table below logged to `stderr` when `--json` is enabled?

I thought duplication is the right approach in the sense that one is for humans, the other for CI (or whatever).

Reply:
yes it will go to stderr, and it will work correctly if you would do `eas command_Name | jq`, but I have never seen json flag used this way in cli tools, it's always replacing human-readable content.

If you feel strongly about it, I'm not a fan of this approach, but I won't object if you decide to go with this solution.

Reply:
Moved everything behind an else statement as you are suggesting. Thanks for explaining you thinking in slack DM's! 🙏